WordPress/src/main/res/values/dimens.xml

```diff
@@ -671,4 +671,16 @@
 
     <dimen name="blogging_reminders_days_top_margin">42dp</dimen>
     <dimen name="blogging_reminders_days_middle_margin">7dp</dimen>
+
+    <!-- theme activation bottom sheet -->
+    <dimen name="theme_activation_bottom_sheet_close_button_size">30dp</dimen>
+    <dimen name="theme_activation_bottom_sheet_selector_size">20dp</dimen>
+    <dimen name="theme_activation_bottom_sheet_divider_height">1dp</dimen>
+    <dimen name="theme_activation_bottom_sheet_margin_8dp">8dp</dimen>
```

Comment:
Using the dimension in the name is generally discouraged. You can also reference reusable values defined above (e.g. `margin_medium`) in that case.